server/plugin/plugin.go

```diff
@@ -90,6 +92,22 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, req *http.R
 	p.http.ServeHTTP(c, w, req)
 }
 
+func (p *Plugin) UserHasBeenCreated(pluginContext *plugin.Context, user *model.User) {
```

Comment:
2/5 We should define the right interface for the `OnXXX` methods (`Hooks`?) and embed it in the Plugin struct directly, avoid these extra proxy methods. Separate PR is ok.

Reply:
I see you have hooks.go in https://github.com/mattermost/mattermost-plugin-apps/pull/13/files.  I'll review and hopefully merge that PR soon and then merge into this branch